src/rocprof-compute

```diff
@@ -38,9 +39,20 @@ try:
     from rocprof_compute_base import RocProfCompute
     from utils.logger import console_error
 except ImportError as e:
-    # print("Failed to import required modules: " + str(e))
-    pass
-
+    # In wheel package, softlink is not supported.
+    # rocprof-compute will get installed in bin and libexec/rocprofiler-compute.
+    # When invoked from bin folder, the extra paths are required to import the dependent scripts
+    try:
+        current_path = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Just want to double check: you expect to use current path, or maybe the root path of ROCm?

Reply:
Expecting current path and use the same to find the relative path 
{current_path}/../libexec/rocprofiler-compute